client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx

```diff
@@ -0,0 +1,15 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+const Superseded: React.FC = () => {
+	const translate = useTranslate();
+
+	return (
+		<p className="display-price__you-own-this">
+			<Gridicon className="display-price__you-own-this-icon" icon="checkmark-circle" size={ 48 } />
+			{ translate( 'Superseded by your plan' ) }
```

Comment:
I have some doubts about the copy here, not being a native speaker. Is that alright? Or would something else feel more natural?

Reply:
This is grammatically correct, but I wonder if Design folks might have some better copy. "Superseded" is probably a bit too advanced of a word, but I couldn't think of anything better when I filed the issue 😅

Reply:
Why are we changing this from "included in your plan"?